Source/UI/NewUI/PaymentForm/Default Implementation/Cardholder/DefaultCardholderFormStyle.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DefaultCardholderFormStyle.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 16/08/2022.
+//
+
+import UIKit
+
+public struct DefaultCardholderFormStyle: CellTextFieldStyle {
+    public var isMandatory: Bool = false
+    public var backgroundColor: UIColor = .clear
+    public var title: ElementStyle? = DefaultTitleLabelStyle(text: Constants.LocalizationKeys.PaymentForm.Cardholder.title)
+    public var hint: ElementStyle?
+    public var mandatory: ElementStyle? = DefaultTitleLabelStyle(
```

Comment:
Question : why is mandatory variable of type `ElementStyle` and then its assigned to a `DefaultTitleLabelStyle`

Reply:
All those labels are conforming to `ElementStyle` which nearly matches an UILabel's behaviour. `DefaultTitleLabelStyle` is an object conforming to `ElementStyle` that is being used for Title kind of Label representation.

... and one of the reasons I was against protocols for this, the complexity is not little 🙈

Reply:
Looking at the code I realised that `mandatory` is used for adding the optional text on a field or not and its style. Which now makes sense, But the naming is bit confusing as we have `isMandatory` and `mandatory` on `CellTextFieldStyle`. 
My understanding is 
`isMandatory` is for responsible to show or hide the mandatory field. 
`mandatory` is for the label style. 

For now we don't need to change anything on the PR, But some comments on these public properties would really help what they are responsible for. ( I dont think this is a POP issue but a improvement in naming )